benefit/VERSION.yaml

```diff
@@ -1 +1 @@
-TAG: 5.12.8
+TAG: 5.13.8
```

Comment:
Please only bump the number on the right e.g. 5.12.9

Reply:
Should the readme be updated then?